InventoryWatch/Countries.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 struct Country: Hashable {
     let name: String
-    
+    let shortcode: String
```

Comment:
there's some redundancy here - the `Countries` dictionary is keyed by each country's `shortcode`. perhaps I could generate the `Countries` dictionary dynamically, keying it by `country.shortcode`?

Reply:
also could use `CountryCode` as the type